### PR TITLE
[ADD] mis_builder_analytic: New module

### PR DESCRIPTION
**Pull request by ernestotejeda**
Cc @Tecnativa TT14942

### CODE REVIEW

**Comment by sbidoul** [comment]
LGTM.

Is there a need for a note in the readme about possible limitations with multiple currencies?

**Comment by sbidoul** [comment]
Travis is red, though.

Also, since the repo is new, we could configure it with the new pre-commit linting?

File: oca_dependencies.txt [hunk 1]
```diff
@@ -1 +1 @@
-mis-builder
+mis-builder https://github.com/Tecnativa/mis-builder 11.0-imp-account_id-mis_builder
```
Comment (pedrobaeza) [inline oca_dependencies.txt]: @ernestotejeda remove this part

Comment (ernestotejeda) [reply oca_dependencies.txt]: It's true, thanks

**Comment by ernestotejeda** [comment]
I'm sorry @sbidoul , I still don't know how to onfigure it with the new pre-commit linting

**Comment by sbidoul** [comment]
@ernestotejeda no problem :) I just pushed the pre-commit config. Can you rebase and run `pre-commit run -a`. More explanations in the [blog post](https://odoo-community.org/blog/the-oca-blog-1/post/black-isort-pre-commit-97).

**Comment by ernestotejeda** [comment]
@sbidoul Thanks for the hint. Changes done

**Review by sbidoul — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**